README.md

```diff
@@ -68,7 +68,7 @@ The library expects a string to be parsed in, other args are optional.
 ```
 string - the string you wish to encode
 
-size   - the size of the qrcode (default 4)
+size   - the size of the qrcode (defaults to smallest size needed to encode the string)
```

Comment:
I wonder if we can use the size (Integer) of the qrcode (defaults to smallest size needed to encode the string) so that the user continues to know the option is a number?

Reply:
```suggestion
size   - the size (Integer) of the qrcode (defaults to smallest size needed to encode the string)
```